scripts/e2e/medusa-be-http.sh

```
@@ -88,8 +88,7 @@ run_tests() {
       MEDUSA_E2E_ADMIN_EMAIL="${MEDUSA_E2E_ADMIN_EMAIL:?MEDUSA_E2E_ADMIN_EMAIL is required}" \
       MEDUSA_E2E_ADMIN_PASSWORD="${MEDUSA_E2E_ADMIN_PASSWORD:?MEDUSA_E2E_ADMIN_PASSWORD is required}" \
       TEST_TYPE="$TEST_TYPE_VALUE" \
-      NODE_OPTIONS=--experimental-vm-modules \
-      pnpm exec jest --runInBand --silent=false ${TEST_TARGET}
+      node ./scripts/run-vitest.mjs run --config vitest.config.ts --no-file-parallelism ${TEST_TARGET}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)